app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,28 @@
+%nav.navbar.navbar-default
+  .container-fluid
+    / Brand and toggle get grouped for better mobile display
+    .navbar-header
+      %button.navbar-toggle.collapsed{"aria-expanded" => "false", "data-target" => "#bs-example-navbar-collapse-1", "data-toggle" => "collapse", :type => "button"}
+        %span.sr-only Toggle navigation
+        %span.icon-bar
+        %span.icon-bar
+        %span.icon-bar
+      %a.navbar-brand{:href => "/"}
+        = image_tag "Instagram_Logo_Large.png" , :class => "brand"
+    / Collect the nav links, forms, and other content for toggling
+    #bs-example-navbar-collapse-1.collapse.navbar-collapse
+      %form.navbar-form.navbar-left{:role => "search"}
+        %input#search.form-control{:placeholder => "Search", :type => "text"}/
```

Comment:
Classes should be listed before IDs (.form-control should precede #search)<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace